NickvisionMoney.Shared/Models/Transaction.cs

```diff
@@ -170,9 +176,10 @@ public object Clone()
             GroupId = GroupId,
             RGBA = RGBA,
             UseGroupColor = UseGroupColor,
-            Receipt = Receipt != null ? Receipt.Clone((x) => { }) : null,
+            Receipt = Receipt?.Clone((x) => { }) ?? null,
             RepeatFrom = RepeatFrom,
             RepeatEndDate = RepeatEndDate,
+            Tags = Tags,
```

Comment:
This needs to be properly cloned. 
```
Tags = Tags.Select(x => x).ToList(),
```

Reply:
`Tags = Tags.Select(x => x).ToList(),`

or 

`Tags = new List<string>(Tags),`

? I guess both works, just want to know if there's a preference